tests/Feature/PetControllerTest.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+namespace Tests\Feature;
+
+use App\Contracts\PetService;
+
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Foundation\Testing\WithFaker;
+
+
+use Tests\TestCase;
+use Mockery\MockInterface;
+
+use App\Models\Pet;
+
+class PetControllerTest extends TestCase
+{
+    private MockInterface $petServiceSpy;
+    private $pets = [];
+
+    private static function getPets(): array
+    {
+        return [
+            Pet::make([
+                'id' => 1,
+                'name' => 'Fido',
+                'age' => 5,
+                'type' => Pet::PET_TYPE_DOG,
+            ]),
+            Pet::make([
+                'id' => 2,
+                'name' => 'Milo',
+                'age' => 3,
+                'type' => Pet::PET_TYPE_CAT,
+            ]),
+        ];
+    }
+
+
+    public function setUp(): void
+    {
+        parent::setUp();
+
+        $this->pets = self::getPets();
+
+        $this->petServiceSpy = $this->spy(PetService::class);
+    }
+
+
+
+
+    public function test_get_Pets_list()
+    {
+        //arrange
+        $this->petServiceSpy->shouldReceive('getPets')
+            ->once()
+            ->andReturn([
+                Pet::make([
```

Comment:
instead of rewriting this every time i'd call your `getPets` method declared at the top